index.html

```diff
@@ -42,8 +42,96 @@ <h3 class="profession">I'am a Software Developer</h3>
+<section class="card1">
+    <div class="work-title grid1"><h1 id="work-title"> My Recent Works</h1></div>
+    <div class="photo1 grid1"><img src="Img/P_img/photo1.png" alt="photo1"></div>
+    <div class="p1-title grid1"><h2 id="p1-title">Multi-Post Stories</h2></div>
+    <div class="p1-text grid1"><p id="p1-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p></div>
+    <div class="list-1 grid1">
+      <ul class="tags1">
+        <li class="flex1">css</li>
+        <li class="flex1">html</li>
+        <li class="flex1">bootstrap</li>
+        <li class="flex1">Ruby</li>
+      </ul>
     </div>
-</div>
+    <div class="button1"><button id="button1">See Project</button></div>
+    </div>
+</section>
+<section class="card2">
+  <div class="p2-title grid2"><h2 id="p2-title">Profesional Art<br>Printing Data</h2></div>
+  <div class="p2-text"><p id="p2-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p></div>
+  <div class="list-2 grid2">
+    <ul class="tags2">
+      <li class="flex2">html</li>
+      <li class="flex2">bootstrap</li>
+      <li class="flex2">Ruby</li>
+    </ul>
+  </div>
+  <div class="button2"><button id="button2">See Project</button></div>
+</section>
+<section class="card3">
+  <div class="p3-title grid3"><h2 id="p3-title">Profesional Art<br>Printing Data</h2></div>
+  <div class="p3-text"><p id="p3-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p></div>
+  <div class="list-3 grid3">
+    <ul class="tags3">
+      <li class="flex3">html</li>
+      <li class="flex3">bootstrap</li>
+      <li class="flex3">Ruby</li>
+    </ul>
+  </div>
+  <div class="button3"><button id="button3">See Project</button></div>
+</section>
+<section class="card4">
+  <div class="p4-title grid4"><h2 id="p4-title">Profesional Art<br>Printing Data</h2></div>
+  <div class="p4-text"><p id="p4-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p></div>
+  <div class="list-4 grid4">
+    <ul class="tags4">
+      <li class="flex4">html</li>
+      <li class="flex4">bootstrap</li>
+      <li class="flex4">Ruby</li>
+    </ul>
+  </div>
+  <div class="button4"><button id="button4">See Project</button></div>
+</section>
+<section class="card5">
+  <div class="p5-title grid5"><h2 id="p5-title">Profesional Art<br>Printing Data</h2></div>
+  <div class="p5-text"><p id="p5-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p></div>
+  <div class="list-5 grid5">
+    <ul class="tags5">
+      <li class="flex5">html</li>
+      <li class="flex5">bootstrap</li>
+      <li class="flex5">Ruby</li>
```

Comment:
- [ ] Kindly remove the border from your technologies so as to improve the design like in the template 
- Your design:
![Annotation 2022-03-18 060408](https://user-images.githubusercontent.com/85113567/158942120-ed4f0217-f5b2-44b1-b30d-2851397e9855.jpg)
- Figma design:
![Annotation 2022-03-18 060420](https://user-images.githubusercontent.com/85113567/158942137-46cf8de8-8adb-4514-9cfb-65ad3c58bce1.jpg)